phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -680,20 +609,21 @@ private void logToIndexToolOutputTable(byte[] dataRowKey, byte[] indexRowKey, lo
             rowKey = new byte[keyPrefix.length];
             Bytes.putBytes(rowKey, 0, keyPrefix, 0, keyPrefix.length);
         }
+        //TODO Write timestamps as strings instead of longs
```

Comment:
Do you still intend to do this TODO and change the longs to strings? If so, why not do it now, and if not, why not get rid of the TODO? :-)

Reply:
I have done with it but forgot to remove the comment. I will remove it